src/js/themes/base.js

```diff
@@ -1317,6 +1317,7 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       container: {
         // elevation: undefined,
         zIndex: '20',
+        height: 'xxsmall',
```

Comment:
Should this be min-height right?
```suggestion
        height: { min: 'xxsmall' },
```

Reply:
Oops didn't see the way the logic was implemented above. Wondering which makes more sense for a consumer?

Reply:
ah yes thats fine

Reply:
makes more sense